org.onionware.onionmedia.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.onionware.onionmedia
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: start.sh
+
+modules:     
+  - name: Desktop
+    buildsystem: simple
+    build-commands:
+      - cp -r . /app
+    sources:
+      - type: archive
+        url: https://github.com/onionware-github/OnionMedia-X/releases/download/2.0.0.0/flatpak-resources.zip
```

Comment:
why doesn't this include the appstream file?

Reply:
I forgot it and will implement, thank you.